cpp/CMakeLists.txt

```diff
@@ -578,6 +579,9 @@ install(DIRECTORY
             ${CUDF_GENERATED_INCLUDE_DIR}/include/libcudacxx
         DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/libcudf)
 
+install(DIRECTORY ${Thrust_SOURCE_DIR}/
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/libcudf/Thrust)
```

Comment:
This feels wrong... We'd presumably have to do this for RMM, cuML, cuGraph, cuSpatial, etc. as well, no? Should we be propagating an install flag somehow to indicate that we should install Thrust when someone uses the install target of libcudf cmake?

Reply:
Or at least in the situation where it goes down the path of adding Thrust since it wasn't found? And presumably we should be doing the same for libcu++?

Reply:
Oh it is absolutely wrong, but @robertmaynard and I decided not to install cuDF's patched version of Thrust in a place where other projects will use it.

Reply:
Unfortunately that makes sense 😞